interactions/base.py

```diff
@@ -6,7 +6,7 @@
     "__authors__",
 )
 
-__version__ = "4.2.1"
+__version__ = "4.3.0-beta.1"
```

Comment:
It's already in unstable..?

Reply:
I have no idea..

Reply:
It is, so why is this showing up? Idk maybe revert that commit and fetch upstream

Reply:
fixed